### ISSUE TITLE
Add Goal button doesn't work

### ISSUE THREAD

**Issue by xriter** [plausible/analytics#4795]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

When going to Site Settings > Goals > Add Goal and then try to submit the form using the 'Add Goal' button, nothing happens.

### Expected behavior

The form should get submitted.

### Environment

```markdown
- OS: MacOS
- Browser: Safari
- Browser Version: 18.1
```

**Comment by ruslandoga** [comment]
👋 @xriter 

Please check your browser console for errors.

**Comment by ruslandoga** [comment]
Hm. Anything in `docker compose logs`? Also note that you need to check the console / logs during or after hitting the "Add Goal" button.

**Comment by ruslandoga** [comment]
Oh, great. Please see https://github.com/plausible/community-edition/wiki/reverse-proxy

tl;dr your reverse proxy needs to have WebSockets enabled. You can also search discussions for more information about this problem (it has been discussed many times).
